express/features/support/steps/shared-functions.ts

```diff
@@ -18,28 +18,36 @@ export async function getButtonLink(page: Page, linkText: string): Promise<Eleme
     return getSingleLink(links, linkText);
 }
 
-export async function clickLink(page: Page, link: ElementHandle, timeout = DEFAULT_TIMEOUT): Promise<any> {
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+export async function clickLink(page: Page, link: ElementHandle, timeout = DEFAULT_TIMEOUT): Promise<unknown> {
```

Comment:
Which warning/error is being ignored here?

Reply:
As I have added some step here. Now this file came in the ES linting radar and starting complaining about the usage of any as data type. Replacing any with never has other consequences and on and on :)

Reply:
![image](https://user-images.githubusercontent.com/111296818/204037467-d141c4a5-97bf-407a-91da-c018efab8bd4.png)

Reply:
ok we will sort this out separately, for now we can ignore those warnings